<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>215</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="originalLabel">
      <property name="text">
       <string>OriginalMesh</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="originalInput">
      <property name="minimumSize">
       <size>
        <width>197</width>
        <height>0</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="originalButton">
      <property name="text">
       <string>Add Selected</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="blendLabel">
      <property name="text">
       <string>BlendShape</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="blendList"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="blendButton">
        <property name="text">
         <string>Add Selected</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delButton">
        <property name="text">
         <string>Remove Selected</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="scaleXRadio">
        <property name="text">
         <string>Mirror X</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="scaleYRadio">
        <property name="text">
         <string>Mirror Y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="scaleZRadio">
        <property name="text">
         <string>Mirror Z</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="runButton">
      <property name="text">
       <string>Run Script</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
